tuna/utils/merge_db.py

```diff
@@ -207,6 +207,16 @@ def best_solver(vals):
 
 def target_merge(master_list, key, vals, keep_keys):
   """merge for explicit target file"""
+  #Error handling of Ivalid input parameter types
+  if key is None:
+    raise ValueError(f'Invalid key value: {key}')
+  if (not (key and not key.isspace())):
+    raise ValueError(f'Invalid key value: {key}')
+  if (not (key and key.strip())):
+    raise ValueError(f'Invalid key value: {key}')
```

Comment:
could simplify to 
```python
if not (key and isinstance(key, str) and key.strip()):
  raise ValueError(f'Invalid key value: {key}')
```
This just asserts that key is a non-empty string. A fairly minimal check.